stdlib/Std/src/Time.luna

```diff
@@ -3,50 +3,165 @@ import Std.Base
 import Std.System
 
 
-# A class representing the time zone.
+# A class representing the timezone.
 # Please keep in mind that the offset is in minutes, as some timezones
 # use the multiple of 15 minutes as the offset.
 class TimeZone:
     TimeZone
     TimeZoneVal Int Bool Text
 
     # Creates a new `TimeZoneInstance`.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    newTimezone = TimeZone.create 60 False "CET"
+    # >    None
+    #
+    # `offsetMins`: Amount of time subtracted from or added to Coordinated Universal Time (UTC) in minutes.
+    # `summerOnly`: Boolean value describing if the timezone has daylight saving time or not.
+    # `name`: Text value contains the three letter shortcut name for this timezone, can be blank.
+    # 
+    # `return`: new timezone instance with given name and offset.
     def create offsetMins summerOnly name:
         TimeZoneVal offsetMins summerOnly name
 
     def toText: case self of
         TimeZoneVal m s n: "TimeZone { minutes = " + m.toText + ", summerOnly = " + s.toText + ", name = " + n + "}"
 
     # The offset (in minutes) from the UTC timezone.
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    min = pacificStandardTime.minutes
+    # >    None
+    #
+    # `return`: Numerical value for the offset in minutes for `self` timezone.
     def minutes: case self of
         TimeZoneVal m _ _: m
 
     # Is this timezone valid only throughout the summertime?
+    #
+    # > import Std.Time 
+    # >
+    # > def main:
+    # >    pacificStandardTime = TimeZone.pst
+    # >    justSummer = pacificStandardTime.summerOnly
+    # >    None
+    #
+    # `return`: Boolean value describing if the timezone has daylight saving time or not.
```

Comment:
This is unclear to me. Is it whether the provided TZ can be in daylight savings time, or whether it currently is? I can see from the code, but the wording is unclear.